### DISCUSSION TITLE
Error in the latest METplus Docker Image

### DISCUSSION THREAD

**Discussion by lindsayrblank** [dtcenter/METplus#1327] [Existing Builds] [answered]
Hello MET folks!

I thought you should know that there is an error in METplus/metplus/wrappers/usage_wrapper.py in the `metplus_latest` Docker image. It has the import statement `from ..util import met_util` instead of `from ..util import LOWER_TO_WRAPPER_NAME`. This is corrected in the metplus-4.1.0-beta4 Docker image and in GitHub `main_v4.0` branch. 

Best,
Lindsay

**Comment by j-opatz** [comment, marked as answer]
Hi Lindsay,

And thanks for bringing this to our attention! George did a little digging into this issue and found that it only occurs when outputting the usage statement, so we haven't created a bugfix release to include the fix. 

As you've stated, this has been corrected in the develop and main_v4.0 branches, so if we do a 4.0.1 bugfix release, or our next major release (4.1.0), this will be resolved.
